test/e2e-upgrade-tests.sh

```diff
@@ -93,6 +93,11 @@ function generate_latest_eventing_manifest() {
   # Download the source code of knative eventing
   git clone https://github.com/knative/eventing.git
   cd eventing
+  if [[ ! -z "$1" ]]; then
```

Comment:
can we add comment above the function to explain that it takes in a parameter, and what the parameter is.

Reply:
sure.